### DISCUSSION TITLE
Show date added on book page

### DISCUSSION THREAD

**Discussion by kanjieater** [gotson/komga#1547] [Ideas]
### Describe your suggested feature

![image](https://github.com/gotson/komga/assets/32607317/90ceb9b0-b1b1-4119-a6eb-95e638ccf322)

Show date, day & time, that a book showed up in a series added/updated here

### Other details

I would like this, as I add series to my library automatically, and want to know if issues were recently added or added long ago. The recently added carousel on the front is the only way I can relatively get this information, but i'd like to see the time something added.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Added both creation and last modified dates
<img width="407" alt="image" src="https://github.com/user-attachments/assets/a0c43734-1cad-458f-a795-7909305187ad" />

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
